src/Look/InfoBoxLook.hpp

```diff
@@ -16,7 +18,8 @@ struct InfoBoxLook {
   bool inverse;
 
   Pen border_pen;
-  Color background_color, focused_background_color, pressed_background_color;
+  Color foreground_color, background_color,
```

Comment:
This new field is never used, is it?

Reply:
True, foreground_color is not used! Will fix that.